examples/with-redux-toolkit/lib/slices/counterSlice.js

```diff
@@ -0,0 +1,31 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const counterSlice = createSlice({
+  name: 'counter',
+  initialState: {
+    value: 0,
+  },
+  reducers: {
+    increment: state => {
+      state.value += 1;
```

Comment:
Let's add a comment suggesting that the reducer is powered by `immer`? Like in:

https://github.com/reduxjs/cra-template-redux/blob/9fe35a6729470305670e8e0d486c72180577661f/template/src/features/counter/counterSlice.js#L10-L13

Reply:
Fixed at 582b278